corehq/apps/sms/tasks.py

```diff
@@ -453,3 +459,10 @@ def sync_phone_numbers_for_domain(domain):
         _sync_case_phone_number(case)
 
     MigrationStatus.set_migration_completed('phone_sync_domain_%s' % domain)
+
+
+def queued_sms():
+    return QueuedSMS.objects.count()
+
+
+datadog_gauge_task('commcare.sms.outbound_queued', queued_sms, run_every=crontab())
```

Comment:
we could rename this to just `queued`, or if we just want the outbound queued we'd need to filter for the direction in the query

Reply:
is this `crontab` meant to be empty?

Reply:
yeah. it means run every minute (same as `minute='*'`). Since it no longer requires a wide time gap for the other metrics it seemed to make the most sense to run it on the minute, as we do with other datadog gauges